### DISCUSSION TITLE
Add custom success message via FORM API

### DISCUSSION THREAD

**Discussion by lucazdj** [elementor/elementor#23114] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I created a custom action to use in some forms. I am trying to rendera a custom message in the form when it is submitted with success, but it is not working.

I am using the function: 

```
public function run( $record, $ajax_handler ) {
...
$ajax_handler->add_success_message( $custom_message );
}
```
but the message rendered in the form is always the default. I also added some error messages, in the same way, and all of them worked.
```
public function run( $record, $ajax_handler ) {
...
$ajax_handler->add_error_message( $custom_message );
}
```

I found this issue about that, but is not resolved
https://github.com/elementor/elementor/issues/9247

### Describe the solution you'd like

It would be nice if this work as intended

**Comment by tjlabais** [comment]
Since there is no update on this ajax handler, you can add a custom message by code using dynamic data then use shortcode. It is dirty, but it works.

**Comment by tjlabais** [comment]
I ended up modifying the class ~/modules/forms/classes/ajax-handler.php for the original method add_success_message to work

	public function send() {
		if ( $this->is_success ) {
            $success_msg = !empty($this->messages['success']) ? implode('<br>', $this->messages['success']) : $this->get_default_message(self::SUCCESS, $this->current_form['settings']);
            wp_send_json_success( [
                'message' => $success_msg,
                'data' => $this->data,
            ] );
		}
